src/sql/workbench/browser/modelComponents/media/card.css

```diff
@@ -66,8 +66,8 @@
 	flex-grow: 1;
 	width: 100%;
 	height: 100%;
-	max-width: 50px;
-	max-height: 50px;
+	max-width: 100px;
```

Comment:
@llali I saw that the ML extension is setting the icon width and height to 100px in modelSourcesComponent.ts. Can you check to make sure it isn't broken now that the max width and height are increased(I don't know how to test it myself)?

Reply:
they do look a bit too big now. I might change the icon size inside the extension after you merge. 
<img width="1000" alt="Screen Shot 2020-11-30 at 1 16 01 PM" src="https://user-images.githubusercontent.com/17187348/100666703-4380b800-330e-11eb-9ee8-6b99eec22b6c.png">

Reply:
No one asked me, but I think the ML extension icon size looks better. 🙃